build.yaml

```diff
@@ -694,13 +811,124 @@ common_cli:
     # /tmp of the slave when the slave is instantiated.
     . /tmp/s3_staging_docs_clusterhq_com.sh
 
+  get_s3_creds_for_bucket_vagrant_jenkins_boxes: &get_s3_creds_for_bucket_vagrant_jenkins_boxes |
+    # collects the S3 credentials that allows us to upload files to the S3
+    # vagrant-jenkins_boxes for our vagrant boxes
+    # the file with the credentials are deployed by the jenkins master to the
```

Comment:
Nit: Either "the file with the credentials is" or "the files with the credentials are". Also in the next clause.